content/ja/handbook/engineering/data-engineering/database-excellence/database-frameworks/index.md

```
@@ -0,0 +1,196 @@
+  - 火曜日（15:00 UTC | EMEA） — [~infradev Issue](https://gitlab.com/groups/gitlab-org/-/issues/?sort=created_date&state=opened&label_name%5B%5D=infradev&label_name%5B%5D=group%3A%3Adatabase%20frameworks&first_page_size=100) で注意が必要なものを確認した後、週次優先事項に集中します。
+  - 木曜日（21:30 UTC | APAC） — 当週の[トリアージ Issue](https://gitlab.com/gitlab-org/database-team/team-tasks/-/issues/?sort=created_date&state=opened&search=Database%20Group%20Triage%20for%20week%20ending&first_page_size=100) を確認し、週次優先事項に集中します。
+- [データベースオフィスアワー](https://docs.google.com/document/d/1wgfmVL30F8SdMg-9yY6Y8djPSxWNvKmhR5XmsvYX1EI/edit?usp=sharing)
+（内部リンク）; [YouTube 録画](https://www.youtube.com/playlist?list=PL05JrBw4t0Kp-kqXeiF7fF7cFYaKtdqXM)
+  - 水曜日、15:30 UTC（隔週）
+  - （APAC）木曜日、3:30 UTC（隔週、交互）
+
+## 作業
+
+GitLab の[エンジニアリングワークフロー](/handbook/engineering/workflow/)ガイドラインに従います。私たちの注意を引くための Issue は、関連プロジェクトに作成してください。`~"group::database frameworks"` ラベルと、その他の関連ラベルを追加してください。
+
+緊急の Issue の場合は、[エンジニアリングマネージャー](#team-members)または Slack チャンネル（[#g_database_frameworks](https://gitlab.slack.com/app_redirect?channel=g_database_frameworks)）に連絡してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)